networkd/network_server.ml

```diff
@@ -595,8 +595,15 @@ module Bridge = struct
 							(debug "%s isn't a valid setting for other_config:vswitch-disable-in-band" dib;
 							None)
 				in
+                let old_igmp_snooping = Ovs.get_mcast_snooping_enable name in
 				ignore (Ovs.create_bridge ?mac ~fail_mode ?external_id ?disable_in_band ?igmp_snooping
-					vlan vlan_bug_workaround name)
+					vlan vlan_bug_workaround name);
+                begin
+                    match old_igmp_snooping, igmp_snooping with
```

Comment:
I'd prefer a simple if statement here actually:

    if igmp_snooping = Some true && not old_igmp_snooping then
      Ovs.inject_igmp_query name;